cli/enry/main.go

```diff
@@ -61,16 +61,22 @@ func main() {
 			return nil
 		}
 
-		content, err := ioutil.ReadFile(path)
-		if err != nil {
-			errors = true
-			log.Println(err)
-			return nil
-		}
-
-		language := enry.GetLanguage(filepath.Base(path), content)
-		if language == enry.OtherLanguage {
-			return nil
+		var language string
+		var ok bool
+		if language, ok = enry.GetLanguageByExtension(path); !ok {
```

Comment:
Actually I can't, if I don't define  the `ok` var out the `if` too, the `:=` in the if statement would define two new variables https://play.golang.org/p/KWI_sbLOaJ